### DISCUSSION TITLE
ダークテーマ搭載

### DISCUSSION THREAD

**Discussion by yukimikudayo** [cuculus-dev/.github#40] [Ideas]
### 要望/アイデアの概要

黒系の背景に白系の文字、という画面表示への切り替え機能

### 利点

明るい背景が苦手な人への配慮、夜間に眩しい画面を避けるなど

**Comment by takecchi** [comment]
@yukimikudayo 自分もダークテーマ信者(IDEもXもMacも)なのでダークテーマはほしいですね！
一応将来的に対応できるような作りにはしてたので、是非やりましょう～
